### PR TITLE
Update revisions error messaging to remove variables

### PR DESCRIPTION
**Pull request by jkmassel**
**To test:**
Let's just make sure that the updated strings look ok in context. I assume they will though, because they'd need to grow / shrink for other languages.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift [hunk 1]
```diff
@@ -303,10 +303,10 @@ private extension Date {
...
     static let noResultsTitle = NSLocalizedString("No history yet", comment: "Displayed when a call is made to load the revisions but there's no result or an error.")
-    static let noResultsSubtitle = NSLocalizedString("When you make changes to your %@ you'll be able to see the history here", comment: "Displayed when a call is made to load the history but there's no result or an error. The placeholder might be 'post' or 'page'.")
+    static let noResultsSubtitle = NSLocalizedString("When you make changes in the editor you'll be able to see the history here", comment: "Displayed when a call is made to load the history but there's no result or an error.")
     static let errorTitle = NSLocalizedString("Oops", comment: "Title for the view when there's an error loading the history")
-    static let errorSubtitle = NSLocalizedString("There was an error loading the %@ history", comment: "Text displayed when there is a failure loading the history. The placeholder might be 'post' or 'page'.")
+    static let errorSubtitle = NSLocalizedString("There was an error loading the history", comment: "Text displayed when there is a failure loading the history.")
 }
```
Comment (danielebogo) [inline WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift]: If you remove a placeholder then you should remove all the _string with format_ too and other logic to display if an abstract post is a post or a page

**Comment by jkmassel** [comment]
Some context: https://wordpress.slack.com/archives/C02RQC4LY/p1542802469060600

As is, the strings will be difficult to translate: https://wordpress.slack.com/archives/C02RQC4LY/p1542744972041900

**Comment by danielebogo** [comment]
@jkmassel if you want i can remove the unused code and push a commit. Translations are good for me though.

**Comment by jkmassel** [comment]
Sure, if it's not too much trouble, thanks!! Sorry for the rushed PR – trying to get this into an 11.3 build going out (hopefully) today 😅

**Comment by danielebogo** [comment]
@jkmassel no problem 😊. I pushed one little commit. If @nheagy can have a quick look and approve this little change I'll be happy!
Also @jkmassel should this PR point to the _release/11.3_?

**Pull request merged**